### PR TITLE
 chore(infra): exclude the examples of check-dependency-version

### PR DESCRIPTION
**Pull request by SoonIter**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,17 +1,12 @@
 {
   "name": "rsdoctor-monorepo",
   "private": true,
-  "workspaces": {
```
Comment (easy1090) [inline package.json:4]: Why delete this?

Comment (SoonIter) [reply package.json:4]: we use `pnpm-workspace.yaml`, i think it is enough

**Review by easy1090 — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (chenjiahan) [reply package.json:4]: changeset will read this field, removing it may break changeset

Comment (SoonIter) [reply package.json]: fixed

**Pull request closed without merging**